src/org/openbravo/erpCommon/ad_forms/AcctServer.java

```diff
@@ -2353,93 +2367,114 @@ public void setMessageResult(ConnectionProvider conn, String _strStatus, String
   /*
    * Sets OBError message for the given status
    */
-  public void setMessageResult(ConnectionProvider conn, VariablesSecureApp vars, String _strStatus,
-      String strMessageType, Map<String, String> _parameters) {
-    String strStatus = StringUtils.isEmpty(_strStatus) ? getStatus() : _strStatus;
+  public void setMessageResult(ConnectionProvider conn, VariablesSecureApp vars, String stringStatus,
+      String strMessageType, Map<String, String> parameters) {
+    String strStatus = StringUtils.isEmpty(stringStatus) ? getStatus() : stringStatus;
     setStatus(strStatus);
-    String strTitle = "";
-    Map<String, String> parameters = _parameters != null ? _parameters
-        : new HashMap<String, String>();
+    Map<String, String> params = parameters != null ? parameters : new HashMap<>();
     if (messageResult == null) {
       messageResult = new OBError();
     }
-    if (strMessageType == null || strMessageType.equals("")) {
+    if (strMessageType == null || StringUtils.isEmpty(strMessageType)) {
```

Comment:
I think that StringUtils already check for null values, so it would be simplier to just use the

StringUtils.isEmpty(strMessageType)